src/Normalizer/DTO.php

```diff
@@ -78,8 +78,8 @@ protected function getAttributeValue($object, $property, $format = null, array $
      * Check to see if we can normalize the object or class
      * {@inheritdoc}
      */
-    public function supportsNormalization($classNameOrObject, $format = null)
+    public function supportsNormalization($classNameOrObject, string $format = null)
     {
-        return $this->entityMetadata->isAnIliosDto($classNameOrObject);
+        return $format === 'jso' && $this->entityMetadata->isAnIliosDto($classNameOrObject);
```

Comment:
typo.

Reply:
More of a, whoops I switched this off and forgot to turn it back on correctly. Thanks.

Reply:
Fixed